addons/web/static/tests/webclient/switch_company_menu_tests.js

```diff
@@ -245,15 +245,15 @@ QUnit.module("SwitchCompanyMenu", (hooks) => {
         assert.containsN(target, "[data-company-id] .fa-square-o", 4);
 
         /**
-         *   [ ] Hermit
+         *   [x] Hermit
          *   [x] **Herman's**     -> log into
          *   [ ] Heroes TM
          *   [ ]    Hercules
          *   [ ]    Hulk
          */
         await click(target.querySelectorAll(".log_into")[1]);
         assert.containsNone(target, ".dropdown-menu", "dropdown is directly closed");
-        assert.verifySteps(["cids=2"]);
+        assert.verifySteps(["cids=2%2C3"]);
```

Comment:
I believe that this is explicitly a behavior that should never happen.
It wasn't like this in 16.0 either.

Reply:
This PR is still very *draft*, but yep my bad, this test shouldn't be modified.